setup.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# setup
+# Setup script for installing btrdb bindings
+#
+# Author:   PingThings
+# Created:  Mon Jan 07 14:45:32 2019 -0500
+#
+# For license information, see LICENSE.txt
+# ID: setup.py [] allen@pingthings.io $
+
+"""
+Setup script for installing btrdb bindings.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import os
+import codecs
+
+from setuptools import setup
+from setuptools import find_packages
+
+##########################################################################
+## Package Information
+##########################################################################
+
+## Basic information
+NAME         = "btrdbextras"
+DESCRIPTION  = "Enhancements additional features to interact with the Berkeley Tree Database"
+AUTHOR       = "Allen Leis"
+EMAIL        = "allen@pingthings.io"
+MAINTAINER   = "Allen Leis"
+LICENSE      = "BSD-3-Clause"
+REPOSITORY   = "https://github.com/BTrDB/btrdb-python"
+PACKAGE      = "btrdb"
+URL          = "http://btrdb.io/"
+DOCS_URL     = "https://btrdb.readthedocs.io/en/latest/"
+
+## Define the keywords
+KEYWORDS     = ('btrdb', 'timeseries', 'database')
+
+## Define the classifiers
+## See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS  = (
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Information Technology',
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: BSD License',
+    'Natural Language :: English',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
+    'Topic :: Database',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+)
+
+## Important Paths
+PROJECT      = os.path.abspath(os.path.dirname(__file__))
+REQUIRE_PATH = "requirements.txt"
+VERSION_PATH = os.path.join(PACKAGE, "version.py")
```

Comment:
This file seems to be missing.

Reply:
agreed.  I havent tried a pip install yet